records_mover/records/pandas/to_csv_options.py

```diff
@@ -108,14 +103,22 @@ def pandas_to_csv_options(records_format: DelimitedRecordsFormat,
             pandas_options['date_format'] = '%m/%d/%y %H:%M:%S.%f'
         else:
             pandas_options['date_format'] = '%m/%d/%y %H:%M:%S.%f%z'
+    elif hints.dateformat == 'DD/MM/YY':
+        if hints.datetimeformattz == hints.datetimeformat:
+            pandas_options['date_format'] = '%d/%m/%y %H:%M:%S.%f'
+        else:
+            pandas_options['date_format'] = '%d/%m/%y %H:%M:%S.%f%z'
+    elif hints.dateformat == 'DD-MM-YY':
+        if hints.datetimeformattz == hints.datetimeformat:
+            pandas_options['date_format'] = '%d-%m-%y %H:%M:%S.%f'
+        else:
+            pandas_options['date_format'] = '%d-%m-%y %H:%M:%S.%f%z'
     else:
         cant_handle_hint(fail_if_cant_handle_hint, 'dateformat', hints)
     quiet_remove(unhandled_hints, 'dateformat')
 
-    # pandas can't seem to export a date and time together :(
```

Comment:
This comment isn't right; it certainly can.  What it can't do is export a time without a date, or a date without a time.  That's what we have prep_for_csv.py for.